<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Garderie</class>
 <widget class="QMainWindow" name="Garderie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>451</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Activit√©</string>
     </attribute>
     <widget class="QLineEdit" name="recherche_activite">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Etablissement</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_activite">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="duree_activite">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_activite">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Duree</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="description_activite">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_activite">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_activite">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_activite">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stat_activite">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>stat duree</string>
       </property>
      </widget>
      <widget class="QComboBox" name="etablissement_activite">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableau_activite">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>551</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="sort_combo_activite">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Tous</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Duree</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Etablissement</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Etablissement</string>
     </attribute>
     <widget class="QComboBox" name="sort_combo_etablissement">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Tous</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Addresse</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tel</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableau_etablissement">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>551</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="id_etablissement">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Addresse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_etablissement">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addresse_etablissement">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_etablissement">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_etablissement">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_etablissement">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="export_excel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>280</y>
         <width>80</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Export Excel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tel_etablissement">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>99 999 999</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="recherche_etablissement">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Chat Bot</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>781</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>781</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
